src/cpp/fastrtps_deprecated/publisher/Publisher.cpp

```diff
@@ -18,7 +18,8 @@
  */
 
 #include <fastrtps/publisher/Publisher.h>
-#include "PublisherImpl.h"
+#include <fastrtps/participant/Participant.h>
```

Comment:
This include can be removed?

Reply:
`src/cpp/fastrtps_deprecated` and `src/cpp/rtps` files were only moved/minimally modified. I believe they are out of scope of this review.